### PR TITLE
Add Zenodo file

### PR DESCRIPTION
**Pull request by sloede**

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        },
+        {
+            "affiliation": "High-Performance Scientific Computing, University of Augsburg, Germany",
+            "name": "Schlottke-Lakemper, Michael",
+            "orcid": "0000-0002-3195-2536"
+        },
+        {
+            "affiliation": "Department of Mathematics; Computational Mathematics, Linköping University, Sweden",
+            "name": "Winters, Andrew R.",
+            "orcid": "0000-0002-5902-1522"
```
Comment (ranocha) [inline .zenodo.json:20]: This should reflect the final list of authors of the paper

File: README.md [hunk 2]
```diff
@@ -93,6 +94,19 @@ where `-f` allows you to provide the path to the control file for which you want
 to create your mesh.
 
 
+## Referencing
+If you use HOHQMesh in your own research, please cite this repository as follows:
+```bibtex
+@misc{kopriva2024hohqmesh,
+  title={{HOHQM}esh: An all Quadrilateral/Hexahedral Unstructured Mesh Generator for High Order Elements},
+  author={Kopriva, David A and Schlottke-Lakemper, Michael and Winters, Andrew R},
```
Comment (ranocha) [inline README.md:102]: Same here

**Comment by ranocha** [comment]
We should also add a similar file to HOHQMesh.jl and use the DOI to cite it in the paper.

File: .zenodo.json (hunk 1 above)
Comment (sloede) [reply .zenodo.json]: Done in aa50993

File: README.md (hunk 2 above)
Comment (sloede) [reply README.md]: aa50993

**Pull request merged**